tests/testlogout.js

```diff
@@ -0,0 +1,31 @@
+import LoginPage from '../pages/login.page';
+import HomePage from '../pages/home.page';
+import LogoutPage from '../pages/logout.page';
+//import { expect } from 'chai';
+
+describe ('TestLogout', async () => {
+
+    it('Should login and logout successfully', async () => {
+
+        await LoginPage.open('login');
+        await LoginPage.loginAs('jsmith@demo.io', 'Demo123!');
+        await expect(HomePage.welcomeUser).toHaveTextContaining('Welcome Josh');
+        
+        await LogoutPage.logout();
+        
+
+    });
+
+    it('Should login successfully and logout pressing the back arrow', async () => {
+
+        await LoginPage.open('login');
+        await LoginPage.loginAs('jsmith@demo.io', 'Demo123!');
+        await expect(HomePage.welcomeUser).toHaveTextContaining('Welcome Josh');
```

Comment:
Same comment about this assertion being unnecessary for the scope of the test case.